pandas/core/indexes/multi.py

```diff
@@ -1263,6 +1263,11 @@ def remove_unused_levels(self):
 
         .. versionadded:: 0.20.0
 
+        Parameters
```

Comment:
Um, are you sure? [Index.set_names](/pandas-dev/pandas/blob/master/pandas/core/indexes/base.py#L1066), [MultiIndex.set_levels](/pandas-dev/pandas/blob/master/pandas/core/indexes/multi.py#L193), and [MultiIndex.set_labels](/pandas-dev/pandas/blob/master/pandas/core/indexes/multi.py#L295) already use inplace. And unlike the latter two functions, `remove_unused_levels` can't change the result of `values`, meaning it should be very safe to use in place since it won't change the semantics of any data structures using the index.